src/stravalib/field_conversions.py

```diff
@@ -37,8 +37,24 @@ def fn_wrapper(field_value: T | None) -> U | None:
 
 @optional_input
 def enum_value(v: ActivityType | SportType) -> str:
+    """A method that collects the root value of a Activity or
```

Comment:
i wondered if we could remove this method because it only returns a single value from a RootModel class which can be easily accessed with .root

Reply:
As we are introducing breaking changes, let's go ahead and return the enum instead of its `str` representation. It improves the accuracy of typing and can potentially prevent bugs for users. Let's document all these breaking changes somewhere.